src/UTF-unknown.csproj

```diff
@@ -17,7 +17,7 @@
   </PropertyGroup>
 
   <ItemGroup Condition=" '$(TargetFramework)' == 'netstandard1.3' Or '$(TargetFramework)' == 'netstandard2.0' ">
-    <PackageReference Include="System.Text.Encoding.CodePages" Version="4.6.0" />
+    <PackageReference Include="System.Text.Encoding.CodePages" Version="4.7.0" />

```

Comment:
It's normal?

Reply:
What's normal?

Reply:
Changing the libraries we depend on is also a change to the API?

Reply:
I don't think there is a problem updating this to 4.7.0